internal/parser/parser.go

```diff
@@ -6495,12 +6495,25 @@ func extractPragmas(commentRange ast.CommentRange, text string) []ast.Pragma {
 		}
 		return []ast.Pragma{pragma}
 	}
-	return nil
 
 	// const singleLine = range.kind === SyntaxKind.SingleLineCommentTrivia && singleLinePragmaRegEx.exec(text);
 	// if (singleLine) {
 	//     return addPragmaForMatch(pragmas, range, PragmaKindFlags.SingleLine, singleLine);
 	// }
+	if commentRange.Kind == ast.KindSingleLineCommentTrivia {
+		matches = singleLinePragmaRegEx.FindStringSubmatch(text)
+	}
+	if commentRange.Kind == ast.KindSingleLineCommentTrivia &&
+		len(matches) > 1 &&
+		(matches[1] == "ts-nocheck" || matches[1] == "ts-check") {
+		return []ast.Pragma{
+			{
+				Name:      matches[1],
+				Args:      make(map[string]ast.PragmaArgument),
+				ArgsRange: commentRange,
+			},
+		}
+	}
 
 	// if (range.kind === SyntaxKind.MultiLineCommentTrivia) {
```

Comment:
This comment too.

Reply:
Or, is this not done? Maybe it's only ported for one pragma?

Reply:
Correct, did not implement multi line